Client/Frontend/Home/TopSites/TopSitesViewModel.swift

```diff
@@ -234,6 +234,7 @@ extension TopSitesViewModel: HomepageViewModelProtocol, FeatureFlaggable {
 // MARK: - FxHomeTopSitesManagerDelegate
 extension TopSitesViewModel: TopSitesManagerDelegate {
     func reloadTopSites() {
+        guard shouldShow else { return }
```

Comment:
Added a check to not reload topsite when they aren't being shown. This is an interim solution that will prevent random reloads of top site section which isn't showing while we implement a proper caching layered solution.